models/r21d/extract_r21d.py

```diff
@@ -15,24 +15,33 @@
 PRE_CENTRAL_CROP_SIZE = (128, 171)
 KINETICS_MEAN = [0.43216, 0.394666, 0.37645]
 KINETICS_STD = [0.22803, 0.22145, 0.216989]
+R21D_MODEL_LIST = [
+    {'model_name': 'default', 'repo': None, 'stack_size': 16 , 'step_size': 16, 'dataset': 'kinetics'},
+    {'model_name': 'r2plus1d_34_32_kinetics', 'repo': 'moabitcoin/ig65m-pytorch',
+        'stack_size': 32 , 'step_size': 32, 'num_classes': 400, 'dataset': 'kinetics'},
+    {'model_name': 'r2plus1d_34_8_kinetics', 'repo': 'moabitcoin/ig65m-pytorch',
+        'stack_size': 8 , 'step_size': 8, 'num_classes': 400, 'dataset': 'kinetics'}
+]
 CENTRAL_CROP_MIN_SIDE_SIZE = 112
-DEFAULT_R21D_STEP_SIZE = 16
-DEFAULT_R21D_STACK_SIZE = 16
 
 class ExtractR21D(torch.nn.Module):
 
     def __init__(self, args):
         super(ExtractR21D, self).__init__()
         self.feature_type = args.feature_type
+        # select R21D model
+        valid_model_names = [m['model_name'] for m in R21D_MODEL_LIST]
+        assert args.model_name in valid_model_names, f'model_name: {args.model_name} not a valid model: {valid_model_names}'
```

Comment:
If `R21D_MODEL_DICT` is a dict, then this part becomes very brief and fluent:

```python
self.model_def = R21D_MODEL_DICT[args.model_name]
```
There would be no need for assert because it will fail with `KeyError` anyway.